### DISCUSSION TITLE
Hay alguna manera de pasarle un bitmap

### DISCUSSION THREAD

**Discussion by abel29** [ImaginativeShohag/Why-Not-Image-Carousel#68] [General]
buenas. soy nuevo en android y quisiera saber si hay alguna manera de pasarle un bitmap a la libreria?

**Comment by ImaginativeShohag** [comment]
No direct way to show the `Bitmap` image object in the Carousel.

But if your image is stored in the drawable folder then you can use them like this:

```kotlin
CarouselItem(
    imageDrawable = R.drawable.image_name
)
```
